okta_sdk/main.tf

```diff
@@ -0,0 +1,152 @@
+  org = "sym-example"
+}
+
+############ General AWS Secrets Manager Setup ##############
+
+# Creates an AWS IAM Role that the Sym Runtime can use for execution
+# Allow the runtime to assume roles in the /sym/ path in your AWS Account
+module "runtime-connector" {
+  source  = "symopsio/runtime-connector/sym"
+  version = ">= 1.0.0"
+
+  # The aws/secretsmgr addon is required to access secrets
+  addons = ["aws/secretsmgr"]
+
+  environment = "main"
+}
+
+# An Integration that tells the Sym Runtime resource which AWS Role to assume
+# (The AWS Role created by the runtime-connector module)
+resource "sym_integration" "runtime_context" {
+  type = "permission_context"
+  name = "main-runtime"
+
+  external_id = module.runtime-connector.settings.account_id
+  settings    = module.runtime-connector.settings
+}
+
+# This resource tells Sym which role to use to access your AWS Secrets Manager
+resource "sym_secrets" "this" {
+  type = "aws_secrets_manager"
+  name = "main-sym-secrets"
+
+  settings = {
+    context_id = sym_integration.runtime_context.id
+  }
+}
+
+############ Okta Integration and Secret Setup ##############
+
+# An AWS Secrets Manager Secret to hold your Okta API Key. Set the value with:
+# aws secretsmanager put-secret-value --secret-id "main/okta-api-key" --secret-string "YOUR-OKTA-API-KEY"
+resource "aws_secretsmanager_secret" "okta_api_key" {
+  name        = "main/okta-api-key"
+  description = "API Key for Sym to call Okta APIs"
+
+  tags = {
+    # This SymEnv tag is required and MUST match the `environment` in your `runtime-connector` module
+    # because the aws/secretsmgr only grants access to secrets tagged with a matching SymEnv value
+    SymEnv = "main"
+  }
+}
+
+# This resource tells Sym how to access your Okta API Key.
+resource "sym_secret" "okta_api_key" {
+  # The source of your secrets and the permissions needed to access
+  # i.e. AWS Secrets Manager, access with IAM Role.
+  source_id = sym_secrets.this.id
+
+  # Name of the key in AWS Secrets Manager
+  path = aws_secretsmanager_secret.okta_api_key.name
+}
+
+# The Okta Integration that your Sym Strategy uses to manage your Okta targets
+resource "sym_integration" "okta" {
+  type        = "okta"
+  name        = "main-okta-integration"
+  external_id = "dev-12345.okta.com"
+
+  settings = {
+    # `type=okta` sym_integrations have a required setting `api_token_secret`,
+    # which must point to a sym_secret referencing your Okta API Key
+    api_token_secret = sym_secret.okta_api_key.id
+  }
+}
+
+############ Basic Approval Flow ##############
+
+resource "sym_flow" "this" {
+  name  = "approval"
+  label = "Approval"
+
+  template       = "sym:template:approval:1.0.0"
+  implementation = "${path.module}/impl.py"
+  environment_id = sym_environment.this.id
+
+  vars = {
+    # we will use this in the Flow implementation to check if the approving user
+    # is authorized to approve this request.
+    approvers_group = "00g4fe7l1jPgGOqzA5d7"
```

Comment:
1. Do we want to make this more clearly an invalid placeholder?
2. Should we link them to [the docs](https://docs.symops.com/docs/okta#add-okta-access-targets) that show how to get the group_id in the comment as well?